gf-ui-lib/src/components/Button/Button.tsx

```diff
@@ -16,6 +16,10 @@ interface ButtonProps {
    * Button type
    */
   type: "submit" | "button";
+  /**
+   * Button size
+   */
+  size?: "content"
```

Comment:
* Please specify button behavior when prop size is not defined in comment. 
* it's wrong external interface. Prop size looks like a flag, but it is not.

Reply:
Done